examples/05-plotting/03-labels.py

```diff
@@ -0,0 +1,54 @@
+"""
+.. _labels:
+
+This example shows how to use nodal labels.
```

Comment:
@ayush-kumar-423 this first line is the title of the example, could you please write something like:
```suggestion
Add Nodal Labels on Plots
```